enzyme/Enzyme/MLIR/Passes/EnzymeBatchPass.h

```diff
@@ -111,6 +111,29 @@ LogicalResult batchOperation(
   return success();
 }
 
+template <typename T>
+FunctionOpInterface batchOperationWithoutInsertingCallOp(
```

Comment:
this should not be necessary

Reply:
yeah but for some reason undoing the first commit(which moves the utils stuff to a new cpp file) kind of messed up compilation